code/utility/logger.cpp

```diff
@@ -11,6 +11,7 @@ void Logger::setStream(QTextEdit* log_stream) {
 
 //TODO: Overload ostream operator
 bool Logger::log (std::string message, LogType type) {
+    if (type == LogType::DEBUG && !Logger::m_debug) return false;

```

Comment:
Not a fan of this implementation. We should use a macro to specify debug mode so it automatically gets set when we compile in debug mode.